src/drivers/boards/px4fmu-v4pro/board_config.h

```diff
@@ -330,7 +328,7 @@ __BEGIN_DECLS
 #define BOARD_ADC_BRICK2_VALID  (px4_arch_gpioread(GPIO_VDD_BRICK2_VALID))
 #define BOARD_ADC_SERVO_VALID   (1)
 #define BOARD_ADC_PERIPH_5V_OC  (px4_arch_gpioread(GPIO_VDD_5V_PERIPH_OC))
-#define BOARD_ADC_HIPOWER_5V_OC (0)
+#define BOARD_ADC_HIPOWER_5V_OC (px4_arch_gpioread(GPIO_VDD_5V_HIPOWER_OC))
```

Comment:
We should check but I would expect that BOARD_ADC_HIPOWER_5V_OC  is positive true logic and 0 is not overcurrent. This is my assumption as when we do not have the signal a 0 was returned.

So wouldn't this be  #define BOARD_ADC_HIPOWER_5V_OC (!px4_arch_gpioread(GPIO_VDD_5V_HIPOWER_OC))